ecosystem/typescript/sdk/src/aptos_client.ts

```diff
@@ -133,14 +137,21 @@ export class AptosClient {
     accountAddress: MaybeHexString,
     query?: { ledgerVersion?: AnyNumber },
   ): Promise<Gen.MoveModuleBytecode[]> {
-    return this.client.accounts.getAccountModules(
-      HexString.ensure(accountAddress).hex(),
-      query?.ledgerVersion?.toString(),
-    );
+    // Note: This function does not expose a `limit` parameter because it might
+    // be ambiguous how this is being used. Is it being passed to getAccountModules
+    // to limit the number of items per response, or does it limit the total output
+    // of this function? We avoid this confusion by not exposing the parameter at all.
+    const f = this.client.accounts.getAccountModules.bind({ httpRequest: this.client.request });
+    const out = await paginateWithCursor(f, accountAddress, 100, query);
```

Comment:
Why not 1000 here? Less IO is preferred.

Reply:
Modules are larger in the response, that was the rationale. We could use 1000 if we want though.

Reply:
Can we make this a constant somewhere?